### PR TITLE
Adding buildpacks example with v1beta1 api

### PR DESCRIPTION
**Pull request by ayushsatyam146**
Adding buildpacks clusterBuildStrategy example to the openshift builds catalog

### CODE REVIEW

**Review by apoorvajagtap — CHANGES_REQUESTED** [review]

File: clusterBuildStrategy/buildpacks/buildpacks.yaml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+spec:
+  volumes:
+    - name: layers-dir
+      emptyDir: {}
+    - name: empty-dir
+      emptyDir: {}
+    - name: cache-dir
+      emptyDir: {}
+  parameters:
+    - name: CNB_PLATFORM_API
```
[the remark this answers is not in the document]
Comment (adambkaplan) [reply clusterBuildStrategy/buildpacks/buildpacks.yaml:15]: I agree that we should keep parameter namings internally consistent. Let's keep `lower-dash-case` as the standard.

**Review by adambkaplan — CHANGES_REQUESTED** [review]

**Comment by apoorvajagtap** [comment]
@ayushsatyam146 can you please update the default value of `builder-image` in [s2i's readme](https://github.com/redhat-developer/openshift-builds-catalog/blob/main/clusterBuildStrategy/source-to-image/README.md#parameters) to `-`. As there's no default for builder-image? (It'd be better to keep this tiny change in a separate commit).

File: clusterBuildStrategy/buildpacks/buildpacks.yaml [hunk 2]
```diff
@@ -0,0 +1,234 @@
+---
+apiVersion: shipwright.io/v1beta1
```
Comment (apoorvajagtap) [inline clusterBuildStrategy/buildpacks/buildpacks.yaml:2]:
```suggestion
apiVersion: shipwright.io/v1alpha1
```
@adambkaplan I just realised that we had to follow the v1alpha1 format for s2i & buildah strategies, and I believe it'll be better to have buildpacks as well per the v1alpha1 version. Thoughts, please?
cc @ayushsatyam146

Comment (apoorvajagtap) [reply clusterBuildStrategy/buildpacks/buildpacks.yaml:2]: I did a quick trial (applying cbs of alpha version), and it appears that the webhook conversion from alpha to beta for this CBS of buildbacks isn't working well. Primary reason being that the [buildpack strategy](https://github.com/shipwright-io/build/tree/main/samples/v1beta1/buildstrategy/buildpacks-v3) upstream is different than this buildpack strategy.

Comment (adambkaplan) [reply clusterBuildStrategy/buildpacks/buildpacks.yaml:2]: There's an upstream version with the [alpha API](https://github.com/shipwright-io/build/blob/main/samples/v1alpha1/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml) - has there been some drift between the two?

**Review by adambkaplan — APPROVED** [review]
/lgtm

Per discussion with @apoorvajagtap - there are issues with the alpha version that are hard to resolve. Since the v1beta1 version works and will be supported long term, let's merge it with this api version.

**Pull request merged**